src/plugins/layout.ts

```diff
@@ -170,15 +171,11 @@ export default class Layout implements IPlugin {
 
   private layoutHandler(element: Element, source: Source) {
     let index = getNodeIndex(element);
+    let nodeInfo = this.shadowDom.getNodeInfo(index);
```

Comment:
>index [](start = 46, length = 5)

Can't index be null? We used to check earlier.. not anymore.

Reply:
If index is null, nodeinfo will be null too

---
In reply to: [182826132](https://github.com/Microsoft/clarity-js/pull/80#discussion_r182826132) [](ancestors = 182826132)